README_shelltest.md

```diff
@@ -0,0 +1,36 @@
+# CLI Testing
```

Comment:
I'm not sure why this hasn't been moved to the tests directory. As I said, these instructions are somewhat helpful for a developer, but I'm not sure a shelltest readme file is necessary in the final product. I would add this to the tests directory, then add an instruction there to run everything from the root directory.

Reply:
Done,
See commit 2efc5a8b842429d88623e0df3209a446d039e37e